test/lib/lbt/calls/SapUiDefine.js

```diff
@@ -1,13 +1,32 @@
 const test = require("ava");
 const {parseJS, Syntax} = require("../../../../lib/lbt/utils/parseUtils");
-
 const SapUiDefineCall = require("../../../../lib/lbt/calls/SapUiDefine");
+const logger = require("@ui5/logger");
+const loggerInstance = logger.getLogger();
+const sinonGlobal = require("sinon");
+const mock = require("mock-require");
 
 function parse(code) {
 	const ast = parseJS(code);
 	return ast.body[0].expression;
 }
 
+test.before((t) => {
```

Comment:
shouldn't this be beforeEach? (same for the other tests)

Reply:
Done